src/analysis/convex/topology.lean

```diff
@@ -0,0 +1,183 @@
+/-
+Copyright (c) 2020 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alexander Bentkamp, Yury Kudriashov
+-/
+import analysis.convex.basic analysis.normed_space.finite_dimension
+
+/-!
+# Topological and metric properties of convex sets
+
+We prove the following facts:
+
+* `convex.interior` : interior of a convex set is convex;
+* `convex.closure` : closure of a convex set is convex;
+* `set.finite.compact_convex_hull` : convex hull of a finite set is compact;
+* `set.finite.is_closed_convex_hull` : convex hull of a finite set is closed;
+* `convex_on_dist` : distance to a fixed point is convex on any convex set;
+* `convex_hull_ediam`, `convex_hull_diam` : convex hull of a set has the same (e)metric diameter
+  as the original set;
+* `bounded_convex_hull` : convex hull of a set is bounded if and only if the original set
+  is bounded.
+-/
```

Comment:
You should mention the topological properties of the standard simplex somewhere in the file level docstring.

Reply:
Done.